modules/_labs/slider/react/README.md

```diff
@@ -0,0 +1,101 @@
+# Canvas Kit React Slider
+
+<a href="https://github.com/Workday/canvas-kit/tree/master/modules/_labs/README.md">
+  <img src="https://img.shields.io/badge/LABS-alpha-orange" alt="LABS: Alpha" />
+</a>  This component is work in progress and currently in pre-release.
+
+slider component
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+or
+
+```sh
+yarn add @workday/canvas-kit-react-slider
```

Comment:
this is an error in our script, but this should read `@workday/canvas-kit-labs-react-slider`